.devcontainer/post-create.sh

```diff
@@ -4,4 +4,10 @@ tar xzvf ./*.tar.gz >/dev/null 2>&1
 sudo mv dependabot /usr/local/bin
 rm ./*.tar.gz
 
+# The image comes loaded with 8.0 SDK, but we need a 7.0 runtime for running tests
```

Comment:
Can you add a clarifying comment explaining why 7.0 is needed for the tests? 

I assume that it is because you target more than one version but it would still be helpful to be explicit

Reply:
done, I'll push up the new comment shortly